### DISCUSSION TITLE
Formatting in Sections with a HUGE Zigbee Network

### DISCUSSION THREAD

**Discussion by jasonhollis** [azuwis/zigbee2mqtt-networkmap#76] [Q&A]
![CleanShot 2024-11-21 at 17 28 20@2x](https://github.com/user-attachments/assets/dcda891f-4395-48cc-90cc-fa388ec486c3)
![CleanShot 2024-11-21 at 17 29 56@2x](https://github.com/user-attachments/assets/39cbc5c5-5c51-4c3f-a32a-79652e283135)
As you can see the network is pretty big and growing.  I'm trying to format the card in HA Sections such that is a bit more useful.  Any help would be greatly appreciated!
![CleanShot 2024-11-21 at 17 31 46@2x](https://github.com/user-attachments/assets/0371426d-f30a-4a94-893d-84a73ec9dfe2)

**Comment by azuwis** [comment]
You may want to take a look at https://nodered.org as https://github.com/azuwis/zigbee2mqtt-networkmap/issues/46 say.

I'm not using Node-Red myself, so don't know about the details.

**Comment by darthrater78** [comment]
Not sure what node red has to do with the question.  I just installed this myself and its almost impossible to use with a large network. 
![image](https://github.com/user-attachments/assets/ee98225d-f315-49c1-9337-419731ab1bcf)

**Comment by darthrater78** [comment]
@jasonhollis I made a new one panel dashboard 

and here's the card:

type: custom:zigbee2mqtt-networkmap
entity: sensor.zigbee2mqtt_networkmap
force: 5500
node_size: 16
height: 1000
width: 1000
view_layout:
  grid-area: full

![image](https://github.com/user-attachments/assets/8eb7e921-a256-4129-8db8-eed89532f77a)

**Comment by azuwis** [comment]
@jasonhollis @darthrater78 Can you please try adding `mqtt_payload: { type: 'raw', routes: false }`:

```
type: custom:zigbee2mqtt-networkmap
entity: sensor.zigbee2mqtt_networkmap
mqtt_payload: { type: 'raw', routes: false }
force: 5500
node_size: 16
height: 1000
width: 1000
view_layout:
grid-area: full
```

and paste the result here?

**Comment by darthrater78** [comment]
![image](https://github.com/user-attachments/assets/03b6b078-1bbb-461f-b1ae-c34de8e2c93a)
![image](https://github.com/user-attachments/assets/81791b22-1823-4234-9340-c2fabbb225cc)

I chose 1500  for with and height since i can scroll and show all the nodes. Many option for this card don't seem to work though. 

I did a refresh afterwards.
